website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -432,6 +432,12 @@ A `default_node_pool` block supports the following:
 
 ~> **Note:** A Route Table must be configured on this Subnet.
 
+* `workload_runtime` - (Optional) Specifies the workload runtime used by the node pool. Possible values are `OCIContainer`.
```

Comment:
I realise this was like this before you opened this PR, but mind making this:

```suggestion
* `workload_runtime` - (Optional) Specifies the workload runtime used by the node pool. The only possible value is `OCIContainer`.
```

to be consistent with other resources, since there's only a single value permitted here?